modules/openapi-generator/src/main/resources/kotlin-spring/libraries/spring-boot/buildGradleKts.mustache

```diff
@@ -0,0 +1,45 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.0.3.RELEASE")
+    }
+}
+
+group = "{{groupId}}"
+version = "{{artifactVersion}}"
+
+repositories {
+    jcenter()
+    mavenCentral()
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions.jvmTarget = "1.8"
+}
+
+plugins {
+    val kotlinVersion = "1.2.41"
+    id("org.jetbrains.kotlin.jvm") version kotlinVersion
+    id("org.jetbrains.kotlin.plugin.jpa") version kotlinVersion
+    id("org.jetbrains.kotlin.plugin.spring") version kotlinVersion
+    id("org.springframework.boot") version "2.0.3.RELEASE"
+    id("io.spring.dependency-management") version "1.0.5.RELEASE"
+}
+
+dependencies {
+    compile("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    compile("org.jetbrains.kotlin:kotlin-reflect")
+    compile("org.springframework.boot:spring-boot-starter-web")
+{{#swaggerAnnotations}}
+    compile("io.swagger:swagger-annotations:1.5.14")
```

Comment:
Is this old swagger-annotations version due to compatibility issues with Spring Boot? Can it be bumped to latest pre-2.0 version (1.5.21)?